st2common/packaging/debian/postinst

```diff
@@ -2,3 +2,4 @@
 
 chmod 755 /usr/bin/st2ctl
 chmod 755 /usr/lib/python2.7/dist-packages/st2common/bin/st2-setup-tests
+mv /etc/logrotate.d/logrotate.conf /etc/logrotate.d/st2.conf
```

Comment:
I like adding it to the makefile...keeps it consistent with the other conf files.